app/assets/stylesheets/forever_style_guide/modules/_nav.scss

```diff
@@ -1,96 +1,108 @@
+  @extend %card-shadow;
 
-      @media(max-width: $grid-float-breakpoint) {
-        @include border_highlight("left", transparent);
-      }
-    }
-    .nav-item-list {
-      @include border_highlight("left", transparent);
-    }
+  .nav.navbar-nav {
+    margin-top: 0;
+    margin-bottom: 0;
 
-    .nav-item-icon {
-      font-size: $font-size-400;
-      line-height: 20px;
-      vertical-align: bottom;
+    @media (max-width: $grid-float-breakpoint) {
+      margin-left: 0;
+      margin-right: 0;
     }
 
-    .nav-item-label {
-      line-height: 20px;
-    }
+    > li > a {
+      height: $header_height;
+      color: color('gray-400');
+      font-size: $font-size-500;
 
-    &:hover, &:focus {
-      .nav-item-tab,
-      .nav-item-list {
-        color: $color-nav-hover;
-        background-color: $color-nav-active;
+      @media (max-width: $grid-float-breakpoint) {
+        padding-top: $offcanvas-padding;
+        @include nav-border(top);
+        border-color: color('gray-600')
       }
-      .nav-item-tab {
-        @include border_highlight("bottom");
 
-        @media(max-width: $grid-float-breakpoint) {
-          @include border_highlight("left");
-        }
+      @media (min-width: $grid-float-breakpoint) {
+        color: color('gray-600');
+        font-size: $font-size-default;
+        @include transition(all 0.2s ease-in-out);
       }
-      .nav-item-list {
-        @include border_highlight("left");
+      &:hover {
+        @media (min-width: $grid-float-breakpoint) {
+          background-color: color('gray-300');
+        }
       }
     }
+    .dropdown.open > .dropdown-toggle {
+      color: color('white');
 
-    @at-root #{&}.active {
-      .nav-item-tab {
-        height: $header_height;
-        color: color('gray-600');
-        background-color: $color-nav-active;
-        @include border_highlight("bottom", color("primary"));
-        border-right: 1px solid color('gray-300');
-        border-left: 1px solid color('gray-300');
+      @media(min-width: $grid-float-breakpoint) {
+        color: color('gray-400');
+        background-color: $nav-bg-color;
+      }
+    }
+    > .active > a {
+      background-color: transparent;
 
+      &:hover,
+      &:focus {
         @media (max-width: $grid-float-breakpoint) {
-          @include border_highlight("left", color("primary"));
-          border-right: 1px solid color('gray-300');
-          border-bottom: 1px solid color('gray-300');
+          color: color('gray-400');
         }
       }
+    }
+    &.navbar-right {
+      margin-right: 0px;
+    }
+  }
+}
 
-      .nav-item-list {
-        color: color('gray-600');
-        background-color: $color-nav-active;
-        @include border_highlight("left", color("primary"));
-        border-top: 1px solid color('gray-300');
-        border-bottom: 1px solid color('gray-300');
-      }
+.navbar-account_item {
+  @media (min-width: $grid-float-breakpoint) {
+    @include nav-border(left);
+  }
+}
 
```

Comment:
I think that return helps separate distinct module styles. Since the .navbar styling is pretty extensive and then the account_item and signed_out-btn are more distinct I think these returns are useful.